boards/arm/efr32_radio/doc/index.rst

```diff
@@ -0,0 +1,173 @@
+.. _efr32_radio:
+
+EFR32 Radio Boards
+##################
+
+.. toctree::
+   :maxdepth: 1
+
+   brd4104a.rst
+   brd4250b.rst
+
+Overview
+********
+
+Support for EFR32 Radio boards is provided by one of the starter kits
+
+- `SLWSTK6020B Bluetooth SoC Starter Kit`_
+- `SLWSTK6000B Mighty Gecko Wireless Starter Kit`_
+- `SLWSTK6061B Proprietary Wireless Starter Kit`_
+
+.. figure:: ./efr32_slwstk6020b.jpg
+   :width: 490px
+   :align: center
+   :alt: EFR32-SLWSTK6020B
+
+   EFR32-SLWSTK6020B (image courtesy of Silicon Labs)
+
+Hardware
+********
+
+Wireless Starter Kit Mainboard:
+
+- Advanced Energy Monitoring provides real-time information about the energy
+  consumption of an application or prototype design.
+- Ultra-low power 128x128 pixel memory LCD
+- 2 user buttons and 2 LEDs
+- Si7021 Humidity and Temperature Sensor
+- On-board Segger J-Link USB and Ethernet debugger
+
+For more information about the EFR32-BRD4001A board, refer to these documents:
+
+- `EFR32BG13 Blue Gecko Bluetooth Starter Kit User's Guide`_
+- `WSTK Main Board BRD4001A Schematics`_
+
+.. _efr32_radio_supported_features:
+
+Supported Features
```

Comment:
It seems that the supported features are currently not the same for the efr32fg and efr32bg.

When i look at the file boards/arm/efr32_radio/efr32_radio_brd4250b.yaml it does not mention the counter, for example.

So i think either this table should be moved to the board specific sub-page or we must ensure that this table is valid for all boards, or we must include some kind of diff or exceptions on the board's sub-page.

Reply:
All the SiLabs SoCs seem to be very uniform. They differ mainly in flash / RAM size and radio capabilities. I updated the .yaml file to add missing features, the tables are aligned. I propose to move data to the radio board pages if/when we find a SoC that breaks the pattern.